### DISCUSSION TITLE
Appreciation for simple design

### DISCUSSION THREAD

**Discussion by BilalBudhani** [pocketbase/pocketbase#74] [Q&A]
I want to take a moment and appreciate how simple and elegant PocketBase's architecture & design is. Unlike other solutions out there PocketBase takes a low-stakes approach with a little learning curve which makes it an easy choice to get started. 

I love the fact that it beautifully packages all the familiar tech (SQLite, REST, Go etc.) putting great features on top (Admin Panel, Multiple auths, File support, Real-Time subscription, JS SDK etc.) with an easy way to deploy while still keeping it extendible. I find this personally remarkable!

I'm learning a lot from the codebase and architecture. I've started using PocketBase already to build a personal invoicing app and I'm liking the DX so far. 

Kudos @ganigeorgiev 🎉

**Comment by jimafisk** [comment]
Just want to +1 this :+1:. I really love the conscious focus on simplicity in an industry that seems to overwhelmingly gravitate towards complexity. It's rare to see such a polished solution that's easy to self host, while still being able to call out to specialized services if desired. @ganigeorgiev hit the nail on the head with this one for me.

**Reply by janzheng** [reply]
Yes agree, I've used Supabase, Firebase, etc. and Pocketbase is BY FAR the best developer experience. Also, your UI for managing data is so simple and intuitive.
Incredible work @ganigeorgiev!!!
